pages/activity/[id]/agenda/[agendaId]/index.tsx

```diff
@@ -95,6 +97,13 @@ export default class AgendaDetailPage extends PureComponent<
             />
           </Col>
         </Row>
+        <Row className="py-5">
+          {!isServer() && (
+            <QRCodeSVG
+              value={`${API_Host}/activity/${this.props.activity.id}/agenda/${this.props.agenda.id}?13800000000`}
```

Comment:
```suggestion
              value={`${API_Host}/activity/${activityId}/agenda/${id}?mobilePhone=13800000000`}
```
1. 在上方解构处取数据，并可重命名。
2. 没参数名怎么取值呢？